examples/text/src/text_selection/index.tsx

```diff
@@ -0,0 +1,147 @@
+// Copyright 2025 The Lynx Authors. All rights reserved.
+// Licensed under the Apache License Version 2.0 that can be found in the
+// LICENSE file in the root directory of this source tree.
+
+import { root } from "@lynx-js/react";
+import { useState } from "react";
```

Comment:
We should import `useState` from `@lynx-js/react`

```suggestion
import { useState } from "@lynx-js/react";
```

Reply:
@colinaaa Done